code/_onclick/ai.dm

```diff
@@ -73,10 +73,6 @@
 	if(world.time <= next_move)
 		return
 
-	if(aicamera.in_camera_mode)
```

Comment:
?

Reply:
Ai camera code was moved out of silicon / cyborg click chain and uses a click intercept now